lib/karafka/setup/validation_schema.rb

```diff
@@ -0,0 +1,43 @@
+module Karafka
+  module Setup
+    # Schema with validation rules for all configuration
+    ValidationSchema = Dry::Validation.Schema do
```

Comment:
All dry validation schemas are validation schemas. I would suggest calling it Karafka::Setup::ConfigSchema (or ConfigurationSchema depending on the existing convention)